ui/desktop/electron-app/src/helpers/platform.js

```diff
@@ -0,0 +1,6 @@
+const os = require('os');
+
+module.exports = {
+  isMac: () => Boolean(os.platform().match(/(darwin)/i)),
+  isWindows: () => Boolean(os.platform().match(/(win32)/i)),
```

Comment:
Silly question. Does all supported versions of windows including 64bits return `win32`?

Reply:
It's a limitation of [os.platform()](https://nodejs.org/api/os.html#os_os_platform). We could use [os.arch()](https://nodejs.org/api/os.html#os_os_arch) in the future, although we don't have a need right now.

Reply:
For sure! not need for it now ;).